.cspell.json

```diff
@@ -83,6 +84,7 @@
         "myroot",
         "mysecretpassword",
         "myvolume",
+        "nogotiation",
```

Comment:
@elenaizaguirre I think we can remove this. Didn't find any occurrence of it via free text search + looks like it has a typo in it.

Reply:
Done, I added it when I was doing different tests and forgot to remove it.

Reply:
Cheers @elenaizaguirre